.gitattributes

```diff
@@ -0,0 +1,63 @@
+###############################################################################
```

Comment:
@rkotti This file has been added at the root of the repo in your fork. I'd suggest removing it from there and then committing again to update the PR